src/main/java/com/immunization/reference/model/PatientDetails.java

```diff
@@ -0,0 +1,249 @@
+package com.immunization.reference.model;
+
+public class PatientDetails {
+
+  private String mrn;
+  private String mrnAuthority;
+  private String nameFirst;
+  private String nameMiddle;
+  private String nameLast;
+  private String nameType;
+  private String motherMaidenNameLast;
+  private String dateOfBirth;
+  private String sex;
+  private String race;
+  private String ethnicity;
+  private String phoneAreaCode;
+  private String phoneNumber;
+  private String addressLine1;
+  private String addressLine2;
+  private String addressCity;
+  private String addressState;
+  private String addressZip;
+  private String addressCountry;
+  private String guardianNameFirst;
+  private String guardianNameLast;
+  private String guardianRelationship;
+
+  private String observationCode;
+  private String observationCodeSet;
+  private String observationResult;
+  private String observationResultSet;
+  private String observationDate;
+
+  public String getMrn() {
```

Comment:
Why are you not using Lombok for this?

Reply:
I forgot it was included in the POM. It's an annotation scheme, right?

Reply:
Yes.

Reply:
done